### ISSUE TITLE
Some items on Admin > Apps page are missing mouse hover focus indicators

### ISSUE THREAD

**Issue by ctrance** [ucla-oarc-mobile/casa-on-rails#212]
@krozett The following are missing mouse hover focus indicators on Admin > Apps page
a.	Show entries dropdown list
 ![image](https://user-images.githubusercontent.com/1034174/108129686-5d0ae380-7063-11eb-9473-f5ebb5050732.png)
b.	Table header sort buttons
c.	Show, edit and destroy links
![image](https://user-images.githubusercontent.com/1034174/108129719-66944b80-7063-11eb-8e50-2c4f4f6f674f.png)

**Comment by ctrance** [comment]
Closing old tickets that we will not fix for now.
